internal/gorson/json/json.go

```diff
@@ -0,0 +1,27 @@
+package json
+
+import (
+	"bytes"
+	"encoding/json"
+	"log"
+)
+
+func Marshal(parameters map[string]string) string {
```

Comment:
this has a different function signature than the stdlib json.Marshal, which will arguably be confusing to our descendants...I really always want to turn a map of strings into a string, though, having this return bytes seems like a bit of a formality.